charts/victoria-metrics-k8s-stack/README.md

```diff
@@ -509,7 +509,7 @@ Change the values according to the need of the environment in ``victoria-metrics
 | vmcluster.ingress.storage.pathType | string | `"Prefix"` |  |
 | vmcluster.ingress.storage.tls | list | `[]` |  |
 | vmcluster.spec.replicationFactor | int | `2` |  |
-| vmcluster.spec.retentionPeriod | string | `"14"` |  |
+| vmcluster.spec.retentionPeriod | string | `"14d"` | Data retention period starting from "1d" (one day) up to "100y" (100 years). See these [docs](https://prometheus.io/docs/prometheus/latest/querying/basics/#time-durations) for the supported duration formats. |
```

Comment:
It has differences with prometheus query duration, like unit `m` is not valid. Please see https://docs.victoriametrics.com/single-server-victoriametrics/#retention.

Reply:
Please update description here, retentionPeriod is not limited by 100y and the link is wrong.

Reply:
@Haleygo is everything fine now?